seed/utils/geocode.py

```diff
@@ -0,0 +1,231 @@
+# !/usr/bin/env python
+# encoding: utf-8
+
+import requests
+import json
+import re
+
+from django.contrib.gis.geos import GEOSGeometry
+from django.db.models import Q
+
+
+class MapQuestAPIKeyError(Exception):
+    """Your MapQuest API Key is either invalid or at its limit."""
+    pass
+
+
+def long_lat_wkt(state):
+    """
+    This translates GIS data saved as binary (WKB) into a text string (WKT).
+    4326 refers to the commonly used spatial reference system and is used
+    for the GIS fields on the PropertyState and TaxLotState models.
+    """
+    if state.long_lat:
+        return GEOSGeometry(state.long_lat, srid=4326).wkt
+
+
+def bounding_box_wkt(state):
+    """
+    This translates GIS data saved as binary (WKB) into a text string (WKT).
+    """
+    if state.bounding_box:
+        return GEOSGeometry(state.bounding_box, srid=4326).wkt
+
+
+def geocode_addresses(buildings):
```

Comment:
Cleanup item - I'm pretty sure I want to rename this method as the name reflects a previous iteration's actual functionality.